distribution/docker/src/docker/Dockerfile

```diff
@@ -38,14 +38,19 @@ RUN <%= retry.loop(package_manager, "${package_manager} update && DEBIAN_FRONTEN
 RUN <%= retry.loop(package_manager, "${package_manager} install -y curl gcc make tar gzip") %>
 <% } %>
 
-<%
+WORKDIR /tmp
+<% if (docker_base == 'iron_bank') {
+// Iron Bank always copies local artifacts
+%>
+COPY zlib-${zlib_version}.tar.gz /tmp
```

Comment:
Perhaps I'm missing something but what actually puts this file here?

Reply:
The Iron Bank Docker repo contains a file, `hardening_manifest.yaml`, that specifies the location of artifacts and their expected checksums. There's a build process somewhere that fetches the files and adds them to the Docker build context.

Reply:
👍 Do we have _any_ automated coverage for building this thing ourselves? If so, does _that_ need to be updatd?

Reply:
We do build and test it, but since we haven't gone to the effort of replicating the Iron Bank environment, we rely on applying some transforms to the `Dockerfile` to make it possible to build locally.